### DISCUSSION TITLE
Cycle only between two window (active and previous-window)

### DISCUSSION THREAD

**Discussion by yaio001** [ianyh/Amethyst#1344] [General]
Hello,
I'm finding difficult to cycle against two window when I have 3 windows open in two-pane layout. I would like to cycle only between the active-window and previous-window, would it it easy to add a custom shortcut for this? Maybe Am I missing something.

**Comment by yaio001** [comment]
Another thing I can't find a way to do would be cycle only in the stack-pane, excluding the master-pane. Sometimes it's boring when I would like to loop selecting window only in the stack pane.

**Comment by yaio001** [comment]
Any guidance welcome, maybe Could I manage it over a custom layout?

**Reply by jottr** [reply]
You can switch focus. I have it set like so in my settings.

<img width="460" alt="image" src="https://user-images.githubusercontent.com/2744198/219983497-68a73445-0385-465c-aa48-d975d1848828.png">

**Comment by ianyh** [comment]
@yaio001 As of 0.18.0 you can alternate focus back and forth from the main window.

**Reply by jottr** [reply]
What's the difference to the "Move focus clockwise" command?

**Reply by ianyh** [reply]
They are equivalent with only two windows, but with more than that it will go back and forth. So you focus the main window, then focusing the main window again will change focus back to the previous window you had focused, then the main window, and so on.

**Reply by jottr** [reply]
Hm, I'm on `0.18.0` and I can't find that option. 

![image](https://user-images.githubusercontent.com/2744198/220368108-aa22943f-62c7-4907-8249-06e33fa60a50.png)
 
![image](https://user-images.githubusercontent.com/2744198/220368151-75ede9ed-ea9e-4117-8e0d-9a06d5de34d6.png)
![image](https://user-images.githubusercontent.com/2744198/220368188-9d1be515-857f-4440-a152-c8f14799ee62.png)
![image](https://user-images.githubusercontent.com/2744198/220368280-29276fee-a354-4c69-888e-43d6b5feabf2.png)

**Reply by ianyh** [reply]
It's the default behavior of moving focus to the main window command.

**Reply by jottr** [reply]
Ok. NowI'm completely confused. Sorry if this is plainly obvious, but I'm failing to grasp this right now. 
What's the _default_ command to move focus?

**Reply by ianyh** [reply]
option + shift + m
